### PR TITLE
Govi Escudero

### PR DESCRIPTION
**Pull request by jaklundun**
### Set up
   - [ ] Use yesterday's project as the base of this project
   - [ ] You _can_ update the HTML/CSS/JS 

### Task
   - [ ] Create 10 _unique_ event listeners
   - [ ]  Use these events to update the DOM in creative ways

### Stretch 
   - [ ] Stretch Task - go look at GSAP
   - [ ] Stretch assignment

### Good habits 
  - [ ] Pull Request title is your first and last name (add any notes in the comment field)
  - [ ] Code has been committed regularly
  - [ ] Commit comments are informative
  - [ ] Code is organized in a logical way

### CODE REVIEW

File: DOM I Repurpose/js/index.js [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+
+//copyright
+
+mainContentP[8].innerHTML = (siteContent['footer']['copyright']);
+
+// DOM II Assignment
+
+// * Create 10 unique event listeners based on the reference file in the objective.  
+
+// 1. changes cta font to red on button click
```
Comment (enitchals) [inline DOM I Repurpose/js/index.js]: Thanks for using comments to call these out -- they're helpful for me, and informative for you or anyone else who wants to refer back to this :smiley:

File: DOM I Repurpose/js/index.js [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+  grabMainCTA.style.fontSize = '10px';
+})
+
+// 3. cta copy is set to 200px on mousing over and out cta copy 
+grabMainCTA.addEventListener('mouseleave', () => {
+  grabMainCTA.style.fontSize = '200px';
+})
+
+// 4. cta copy is set to 10px when window object is resized 
+window.addEventListener('resize', () => {
```
Comment (enitchals) [inline DOM I Repurpose/js/index.js]: Great work mixing and matching these event listeners!

File: DOM I Repurpose/js/index.js [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+
+grabMiddelImg.addEventListener('contextmenu', () => {
+  alert(`Don't even think about lifting this image!`);
+})
+
+
+// 7.  if you copy anything from page and then try to paste it, the copied content will not appear, rather , a special message will appear instead.
+
+document.addEventListener('copy', function(e){
+  e.clipboardData.setData('text/plain', 'Hello, Mr. Chris Honda');
```
Comment (enitchals) [inline DOM I Repurpose/js/index.js]: This is SO COOL. I had no idea you could do this. Great work!

File: DOM I Repurpose/js/index.js [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+let pageContainer = document.querySelector('.container')
+
+window.addEventListener('scroll', () => {
+  pageContainer.style.backgroundColor = "violet"
+});
+
+
+// 10. Upon double clicking anywhere in the document the background will turn white
+
+document.addEventListener('dblclick', () => {
```
Comment (enitchals) [inline DOM I Repurpose/js/index.js]: I like that you gave me a way to effectively undo another event listener's changes.

**Comment by jaklundun** [comment]
Thank you, Ellen!

**Pull request closed without merging**